fonts.css

```diff
@@ -0,0 +1,27 @@
+@font-face {
+  font-family: 'ptserif';
+  src: url('ptserif-bold-webfont.woff') format('woff');
+  font-weight: bold;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: 'ptserif';
+  src: url('ptserif-bolditalic-webfont.woff') format('woff');
+  font-weight: bold;
+  font-style: italic;
+}
+
+@font-face {
+  font-family: 'ptserif';
+  src: url('ptserif-italic-webfont.woff') format('woff');
+  font-weight: normal;
+  font-style: italic;
+}
+
+@font-face {
+  font-family: 'ptserif';
+  src: url('ptserif-regular-webfont.woff') format('woff');
+  font-weight: regular;
+  font-style: normal;
+} 
```

Comment:
Codacy found an issue: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/LessonsHTML/pullRequest?prid=6226470)